src/RegistryItemOptionsResolver.php

```diff
@@ -23,5 +27,27 @@ public function __construct(array $options)
 				break;
 		}
 
+		if (null !== $options['metadata']) {
+			foreach (preg_split('/,?\s+/', $options['metadata']) as $name) {
```

Comment:
uhmm ... yeah, my bad for the regex, but as for the rest - I wouldn't be so enforcing in terms of  space char :)